.github/workflows/pipeline-e2e-test.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
         subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
         resource-group: open-${{ secrets.RESOURCE_GROUP_TMP }}
         deployment-file-path: ./mlops/bicep/open_sandbox_setup.bicep
+        deployment-name: open_sandbox_${{ github.run_id }}
```

Comment:
Can we change this `open_sandbox_${{ github.run_id }}` to `open_sandbox_{{ github.run_attempt }}${{ github.run_id }}` (everywhere)? coz if we re-run any pipeline directly through the Actions tab, then the GitHub run id remains the same, only the github.run_attempt changes.